sycl/include/CL/sycl/detail/pi.h

```diff
@@ -305,6 +305,7 @@ typedef enum {
   PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE = 0x10025,
   PI_DEVICE_INFO_MAX_MEM_BANDWIDTH = 0x10026,
   PI_DEVICE_INFO_IMAGE_SRGB = 0x10027,
+  PI_DEVICE_INFO_HOMOGENEOUS_ARCH = 0x10028,
```

Comment:
Please add comment, explaining its purpose and type of data.
Also I think the name is misleading. Maybe PI_DEVICE_INFO_BUILD_ON_SUBDEVICE, return true if sub-device should do its own build.

Reply:
Renamed and added a comment as suggested.